ProjectFiles/Assets/Editor/ServerBuild.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEditor;
+using UnityEditor.Build.Reporting;
+using UnityEngine;
+
+
+public class ServerBuild
+{
+    
+    [MenuItem("Build/Build Server")]
+    public static void BuildServer()
+    {
+        BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
+        buildPlayerOptions.scenes = new[] { "Assets/Scenes/ModelShowcase.unity" };
+        
+
+        const string buildFolder = "../builds/server/server";
+        
+        buildPlayerOptions.locationPathName = buildFolder;
+
+        buildPlayerOptions.target = BuildTarget.StandaloneLinux64;
+        buildPlayerOptions.options = BuildOptions.EnableHeadlessMode;
+        
+        BuildReport report = BuildPipeline.BuildPlayer(buildPlayerOptions);
+        BuildSummary summary = report.summary;
+
+        if (summary.result == BuildResult.Succeeded)
+        {
+            Debug.Log("Build succeeded: " + summary.totalSize + " bytes");
+        }
+
+        if (summary.result == BuildResult.Failed)
+        {
+            Debug.Log("Build failed");
+        }
+    }
+    
+    [MenuItem("Build/Build Client")]
+    public static void BuildClient()
+    {
+        BuildPlayerOptions buildPlayerOptions = new BuildPlayerOptions();
+        buildPlayerOptions.scenes = new[] { "Assets/Scenes/ModelShowcase.unity" };
```

Comment:
Should they not build the same scenes?

Reply:
They both have that scene because the server simulates the players in that scene too

Reply:
Ignore me, I was thinking you were duplicating the same build options but they are different for each, I just wondered if there was a way of both giving them the same scenes without manually giving them to the client and the server.

Reply:
Ohh yeah I might tweak that at some point.